command/job_status.go

```diff
@@ -516,11 +520,10 @@ func formatJobActions(actions []string) string {
 	actionsOut[0] = "Action Name|Task Group|Task"
 
 	for i, action := range actions {
-		parts := strings.Split(action, "|")
-		if len(parts) != 3 {
+		group, task, actionName := action["group"], action["task"], action["action"]
+		if group == "" || task == "" || actionName == "" {
```

Comment:
seems weird these may be empty. and could any of them be empty without all of them being empty?

Reply:
You know, I think group/task can never be empty, and we explicitly guard against actionName being empty too. Maybe this is an unreachable condition?